PFS-IPMU-00001/infra.rst

```diff
@@ -0,0 +1,85 @@
+Infrastructure overview (storage, VM host)
+======
+
+ToC
+
+* `Storage service configuration`_
+* `VM host configuration`_
+
+Storage service configuration
+------
+
+Two categories of storage services are provided, one is actively used service 
+backend, and another is archive and/or backup service backend. 
+
+For actively used service backend, storage is exposed through VM guest which 
+mounts physical drive through virtio. 
+Disk drives are placed in one SAS expander box and controlled by RAID card at 
+host computer as RAID1 with hot spare. On hardware issue, it is possible to 
+move disk drives into another host computer to continue services. 
+To survive UPS failure, both SAS expander box and host computer shall be 
+connected to two separated UPS as redundant power source. 
+Four RAID1 are mounted and exposed by three VM guests: 
+
+* pd (pfsdisk): /virt and /server
+* pc (pfscalc): /data1
+* sd (simdata): /simdata
+
+
+For archive service backend, we have one iSCSI connected RAID6 storage server 
+and two VM guests to host storage via iSCSI. 
+The storage server is not easily replacable, and need to consult with vender 
+on hardware issue. Disk drives are configured as RAID6 with hot spare. 
+Two VM guests are for: 
+
+* PFS development data archive (pfsarch)
```

Comment:
It will be better to emphasize the name of VM guest by changing format for clarify.... this is just thought.

Comparing the above paragraph, it looks confusing a little which is the VM guest name, and which dn.

Reply:
+1